Example/Database/Tests/Unit/FSyncPointTests.m

```diff
@@ -15,9 +15,7 @@
  */
 
 #import <Foundation/Foundation.h>
```

Comment:
System imports/includes should be in a separate group

Reply:
Fixed.